RELEASE/scripts/autoscend/auto_providers.ash

```diff
@@ -146,6 +146,19 @@ float providePlusCombat(int amt, location loc, boolean doEquips, boolean specula
 		return result();
 	}
 
+	if (!speculative)
+	{
+		//Prep for if other +combat familiars are added
+		foreach fam in $familiars[Jumpsuited Hound Dog]
+		{
+			if(auto_have_familiar(fam))

```

Comment:
should also check for `canChangeToFamiliar` for QT?

also check `pass()` before returning result -- doesn't matter at the moment because it just returns result() anyway but would be relevant if anything got added below this entry?